README.md

```diff
@@ -1,19 +1,20 @@
 # CF Notify
 
 ## What?
-An AWS Lambda function that will post Cloud Formation status updates to a Slack channel via a Slack Web Hook.
+An AWS Lambda function that will post Cloud Formation status updates to a Slack channel via a Slack Web Hook. Additionally it will notify you of stacks that do not have notifications enabled.
```

Comment:
I think lambdas should follow the single responsibility principle as it makes things easier to reason about but also because lambdas have a limited execution time, so doing one thing per lambda is advised.

Step Functions are a great way to orchestrate a workflow of lambdas, so maybe that's an option here. https://aws.amazon.com/step-functions/

However, I would think AWS Config Rules would be a better fit for this problem. https://aws.amazon.com/blogs/aws/aws-config-rules-dynamic-compliance-checking-for-cloud-resources/

Reply:
I'll close this PR but FWIW there is a cost associated with creating a separate Lambda for every responsibility. Another Git repo, another project that has to be maintained, more CI/CD, more infra automation, another IAM role, etc. This was a minor addition that does not add much complexity to the project. Also, AWS Config Rules does not natively support pushing alerts to Slack. This has all of the boilerplate code to handle that.